src/main/resources/db/changelog/v1.0.0/2-create-cart_entry-table.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.17.xsd">
+
+    <changeSet author="GeorgeT88" id="2022-11-28-19.02">
+
+        <preConditions onFail="HALT">
+            <not>
+                <tableExists schemaName="cart" tableName="cart_entry"/>
+            </not>
+        </preConditions>
+
+        <createTable schemaName="cart" tableName="cart_entry">
+            <column name="id" type="bigint" autoIncrement="true" >
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="product_name" type="varchar(100)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="quantity" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="cart_id" type="bigint">
+                <constraints nullable="false"/>
```

Comment:
we need a foreign constraint to cart id column

Reply:
Done! good remark! 👍

Reply:
added this change to Order aswell